src/react-app-env.d.ts

```diff
@@ -0,0 +1 @@
+/// <reference types="react-scripts" />
```

Comment:
What is this file supposed to be?

Reply:
autogenerated whenever i run npm run start - something from cra